data-connect/flutter-blank/lib/main.dart

```diff
@@ -56,8 +58,20 @@ class ShowError extends StatelessWidget {
       ),
       home: const Scaffold(
           body: SafeArea(
-              child: Text(
-                  "Open a new terminal below by clicking +. Run \"flutterfire configure -y -a com.example.blank\" to continue and refresh the page."))),
+              child: Center(
```

Comment:
You can also do SelectableArea without needing to do it on each of the Text btw

Reply:
Updated. Ready to merge whenever!